P0009/P0009.bs

```diff
@@ -2267,34 +2232,21 @@ static constexpr index_type static_extent(size_t r) noexcept;
 <br/>
 
 ```c++
-constexpr Extents get_extents() const noexcept;
+constexpr const extents_type & extents() const noexcept;
 ```
 
-* *Returns:* `get_mapping().get_extents()`.
+* *Returns:* `mapping().extents()`.
 
 <br/>
 
 ```c++
 constexpr index_type extent(size_t r) const noexcept;
 ```
 
-* *Returns:* `get_extents().extent(r)`.
+* *Returns:* `extents().extent(r)`.
 
 <br/>
 
-```c++
-constexpr index_type size() const noexcept;
```

Comment:
We believe we need size and unique_size. One example is I want to pack a strided view into an MPI buffer. I need to know how large to make that buffer, and its not required_span_size. And if we add size we probably also need a function which returns something discounting all the replicated entries in the mapping.

Reply:
Pull them from the mapping :-)

Reply:
The reasoning for having `size()` and `unique_size` here is for consistency with container-like types (this is going into the containers subclause of the standard after all) in the case of `size()`, and for clarity as to which size the `size()` method refers to in the case of `unique_size`.